packages/algob/src/lib/msig.ts

```diff
@@ -89,3 +89,31 @@ export function validateMsig (msig: MultiSig | undefined): void {
     throw new Error("Error fetching multisigned logic signature from file - invalid/undefined msig");
   }
 }
+
+/**
+ * Description: Signs an encoded signed multi-sig transaction object
+ * @param {Account} signerAccount
```

Comment:
```suggestion
 * @param signerAccount
```
We don't need to add type in doc string, as in typescript we already added types in function arguments.

Reply:
okay